src/Microsoft.Diagnostics.Runtime/src/Utilities/PEImage/PEImage.cs

```diff
@@ -76,6 +79,10 @@ private PEImage(Stream stream, bool leaveOpen, bool isVirtual)
             if (!stream.CanSeek)
                 throw new ArgumentException($"{nameof(stream)} is not seekable.");
 
+            _memoryStream = new MemoryStream();
```

Comment:
This will blow up memory usage.  We will now have an in memory copy of every PEImage we load/read from.  This also won't work well with ReadVirtualStream, which doesn't always know the true ending of a partial image in memory, especially in the minidump case.